src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,19 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+       Bulldozer bulldozer = new Bulldozer();
+       Excavator excavator = new Excavator();
+       Truck truck = new Truck();
```

Comment:
[Checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop): Use abstract reference instead of specific one where possible: